packages/@react-aria/focus/src/FocusScope.tsx

```diff
@@ -460,6 +460,7 @@ function useRestoreFocus(scopeRef: RefObject<HTMLElement[]>, restoreFocus: boole
         document.removeEventListener('keydown', onKeyDown, true);
       }
 
+      // eslint-disable-next-line react-hooks/exhaustive-deps
```

Comment:
shouldn't this be down on line 471?

Reply:
The `scopeRef.current` within the `isElementInScope(document.activeElement, scopeRef.current)` call on line 464 is what triggers the eslint warning:

```ts
      if (restoreFocus && nodeToRestore && isElementInScope(document.activeElement, scopeRef.current)) {
```